pkgs/top-level/all-packages.nix

```diff
@@ -40537,6 +40537,19 @@ with pkgs;
 
   nix = nixVersions.stable;
 
+  # The aws-sdk-cpp tests are flaky.  Since pull requests to staging
+  # cause nix to be rebuilt, this means that staging PRs end up
+  # getting false CI failures due to whatever is flaky in the AWS
+  # SDK tests.  Since none of our CI needs to (or should be able to)
+  # contact AWS S3, let's just omit it all from the Nix that runs
+  # CI.  Bonus: the tests build way faster.
```

Comment:
Please include a link to the upstream issue: https://github.com/NixOS/nix/issues/7582